MaterialSkin/HTML/material/html/lang/ru.json

```diff
@@ -374,35 +374,35 @@
 "%1 Years":"%1 Годов",
 "1 day":"1 день",
 "%1 days":"%1 дней",
-"Add URL to queue":"",
-"Add favorite, podcast, or preset":"",
-"Adding albums...":"",
-"Adding tracks...":"",
-"Advanced":"",
-"Advanced search":"",
-"Alt+%1":"",
-"Alt+N":"",
-"Appended all tracks to the play queue":"",
-"Appended track to the play queue":"",
-"Clear queue":"",
-"Configure browse modes":"",
-"Ctrl+%1":"",
-"Ctrl+Shift+%1":"",
-"Decrease volume":"",
-"Don't Stop The Music":"",
-"Enable keyboard shortcuts":"",
-"Go back":"",
-"Go to homescreen":"",
-"Increase volume":"",
-"Keyboard shortcuts":"",
-"Loading...":"",
-"Next track":"",
-"Play/pause":"",
-"Previous track":"",
-"Save queue":"",
-"Scroll queue to current track":"",
-"Spacebar":"",
-"Switch to Nth player":"",
-"⌘+%1":"",
-"⌘+Shift+%1":""
+"Add URL to queue":"Добавить URL в очередь",
+"Add favorite, podcast, or preset":"Добавить избранное, подкаст или предустановленные",
+"Adding albums...":"Добавить альбомы",
+"Adding tracks...":"Добавить треки",
+"Advanced":"Расширенный",
+"Advanced search":"Расширенный поиск",
+"Alt+%1":"Alt+%1",
```

Comment:
For strings that are the same, you can just leave blank. This is how I handle the British English translation - I only add translations where the spelling differs.